### PR TITLE
Refactor invoices.update

### PR DESCRIPTION
**Pull request by luisabritotl**
- Remove department_id 
- Add nullable option to some fields

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2223,21 +2223,20 @@ Update an existing invoice.
...
             + type (enum[string])
                 + Members
                     + cash - Direct payment, often cash
                     + end_of_month - End of the month of Xth day after the invoice date
                     + after_invoice_date - X days after invoice date
             + days (number, optional) - Modifier "X" for the above statements. Not required when type is 'cash'.
-        + layout_id: `8c5f1a13-12ac-493f-9d82-25af7952c351` (string, optional)
-        + purchase_order_number (string, optional) - The external order number. Overrides possible PO number coming from project.
-        + remarks (string, optional) - Free form text. Visible on the invoice depending on the template.
+        + layout_id: `8c5f1a13-12ac-493f-9d82-25af7952c351` (string, optional, nullable) - If null will reset to the default one.
```
Comment (stivni) [inline apiary.apib]: PP: I'm not sure about this behavior... can we keep this mandatory if the property is provided?
It's easier to make it nullable in the future than it is to make it non-nullable if it is now nullable :)

**Comment by stivni** [comment]
@luisabritotl is this PR still relevant?

**Comment by luisabritotl** [comment]
I guess not, at some point it was a reference

**Comment by luisabritotl** [comment]
No longer using this documentation, so it will be closed

**Pull request closed without merging**